lib/vanagon/platform/windows.rb

```diff
@@ -44,6 +47,16 @@ def generate_packaging_artifacts(workdir, name, binding)
         ["chocolateyInstall.ps1", "chocolateyUninstall.ps1"].each do |win_file|
           FileUtils.copy(File.join(VANAGON_ROOT, "resources/windows/nuget/#{win_file}"), File.join(workdir, win_file))
         end
+
+        # Now for the msi (Wix) artefacts
```

Comment:
Lets compromise - keep the UK spelling for the comment and the US one for the function name :)